### PR TITLE
Global ID fixing order of the conditional

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#71152](http://central.tri.be/issues/71152)
---

### CODE REVIEW

**Comment by barryhughes** [comment]
:sun_with_face:

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $meta_id is never used | [Tracker.php:109](https://github.com/moderntribe/tribe-common/blob/fb7405743aedec2614c564ae7c2d4d422c6af626/src/Tribe/Tracker.php#L109)
:warning: | The method parameter $meta_value is never used | [Tracker.php:109](https://github.com/moderntribe/tribe-common/blob/fb7405743aedec2614c564ae7c2d4d422c6af626/src/Tribe/Tracker.php#L109)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $meta_id is never used | [Tracker.php:109](https://github.com/moderntribe/tribe-common/blob/7091b017ad8e1eec98fb0f43ff6cdc85d004d638/src/Tribe/Tracker.php#L109)
:warning: | The method parameter $meta_value is never used | [Tracker.php:109](https://github.com/moderntribe/tribe-common/blob/7091b017ad8e1eec98fb0f43ff6cdc85d004d638/src/Tribe/Tracker.php#L109)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
Ready for another pair of  👀

File: src/Tribe/Tracker.php [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+		// Fetch the current data from the modified fields
+		$modified = get_post_meta( $post->ID, self::$field_key, true );
+		if ( ! is_array( $modified ) ) {
+			$modified = array();
+		}
+
+		// If we got here we will update the Modified Meta
+		$modified[ $meta_key ] = $now;
+
+		bdump( $modified );
```
Comment (borkweb) [inline src/Tribe/Tracker.php]: This sounds unpleasant.

File: src/Tribe/Tracker.php [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+	 * @param null|bool $check      Whether to allow updating metadata for the given type.
+	 * @param int       $post_id    Post ID.
+	 * @param string    $meta_key   Meta key.
+	 * @param mixed     $meta_value Meta value. Must be serializable if non-scalar.
+	 * @param mixed     $prev_value Previous Value of the Meta, allowing to check the data
+	 *
+	 * @return null|bool            This should be ignored, only used to not break the WordPress filters
+	 */
+	public function filter_watch_updated_meta( $check, $post_id, $meta_key, $meta_value, $prev_value = null ) {
+		bdump( $meta_key );
```
Comment (borkweb) [inline src/Tribe/Tracker.php]: `bdump`? Sounds disgusting.

**Comment by borkweb** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $meta_id is never used | [Tracker.php:109](https://github.com/moderntribe/tribe-common/blob/7091b017ad8e1eec98fb0f43ff6cdc85d004d638/src/Tribe/Tracker.php#L109)
:warning: | The method parameter $meta_value is never used | [Tracker.php:109](https://github.com/moderntribe/tribe-common/blob/7091b017ad8e1eec98fb0f43ff6cdc85d004d638/src/Tribe/Tracker.php#L109)

> via jenkins/codesniffer-phpcs

**Comment by borkweb** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $meta_id is never used | [Tracker.php:109](https://github.com/moderntribe/tribe-common/blob/7091b017ad8e1eec98fb0f43ff6cdc85d004d638/src/Tribe/Tracker.php#L109)
:warning: | The method parameter $meta_value is never used | [Tracker.php:109](https://github.com/moderntribe/tribe-common/blob/7091b017ad8e1eec98fb0f43ff6cdc85d004d638/src/Tribe/Tracker.php#L109)

> via jenkins/codesniffer-phpcs

**Pull request merged**